scripts/page/page-template.js

```diff
@@ -0,0 +1,18 @@
+const { toPascalCase } = require("../utilities");
+
+const createPageTemplate = handle => {
+  const componentName = toPascalCase(handle);
+  return `import { Heading, Page } from "@shopify/polaris";
+
+const ${componentName} = () => (
+  <Page>
+    <Heading>You created a new page called ${componentName}</Heading>
+    <p>You can visit this page by appending "/${componentName}" to your URL</p>
```

Comment:
what about also something like `you can edit this page in #{file_path}`?

Reply:
Oh good one!